common/persistence/cassandra/mutable_state_store.go

```diff
@@ -892,6 +902,7 @@ func (d *MutableStateStore) assertNotCurrentExecution(
 			State:            enumsspb.WORKFLOW_EXECUTION_STATE_UNSPECIFIED,
 			Status:           enumspb.WORKFLOW_EXECUTION_STATUS_UNSPECIFIED,
 			LastWriteVersion: 0,
+			StartTime:        startTime,
```

Comment:
We can get the value from `resp` I think. (The passed in startTime can also be empty)
Not sure why we are not populating other fields, but looks like we can do the same for other fields as well.

Reply:
I'm not going to change that, because I think we have a bigger problem. 
assertNotCurrentExecution is calling GetCurrentExecution, and GetCurrentExecution is making a DB query.
Which is not needed - all information at that point should be available.
Lets discuss it

Reply:
I take it back.
it is used in UpdateWorkflowExecution, which does NOT necessary update current workflow.